src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
```

Comment:
You don't need to use React fragment here

Reply:
Agree with Iryna, fragment is redundant here